modules/client/src/internal/constants.ts

```diff
@@ -7,3 +8,9 @@ export const AVAILABLE_FUNCTION_SIGNATURES: string[] = [
   DAO__factory.createInterface().getFunction("grant").format("minimal"),
   DAO__factory.createInterface().getFunction("freeze").format("minimal"),
 ];
+
+export const UNSUPPORTED_DAO_METADATA_LINK: IMetadata = {
```

Comment:
Same suggestion, metadata fetching could fail because something is unpinned for example